docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -245,6 +245,7 @@ As part of the customization API, the grid allows you to replace and override ne
 - `noRowsOverlay`: The component rendered when the rows react prop is empty or [].
 - `footer`: The component rendered below the viewport.
 - `pagination`: The component rendered for the pagination feature.
+- `toolbar`: The component rendered above the column header bar and the custom `header` component if provided.
```

Comment:
They are sorted by vertical display order. I assume it should be before `header`

Reply:
done